lighthouse-treemap/app/src/main.js

```diff
@@ -156,17 +194,20 @@ class TreemapViewer {
    * @param {LH.Treemap.Node} node
    */
   makeCaption(node) {
-    const size = node.resourceBytes;
-    const total = this.currentTreemapRoot.resourceBytes;
+    const partitionBy = this.currentViewMode.partitionBy || 'resourceBytes';
+    // @ts-expect-error
```

Comment:
can we fix this with better types on`ViewMode`? `partitionBy: 'resourceBytes' | 'unusedBytes'`